decisions/models/case.py

```diff
@@ -50,15 +53,16 @@ def __str__(self):
 
 
 class Action(DataModel):
-    title = models.CharField(max_length=255,
-                             help_text=_('Title of the action'))
+    translations = TranslatedFields(
+        title=models.CharField(max_length=255, help_text=_('Title of the action')),
+        resolution=models.CharField(max_length=255, blank=True,
+                                    help_text="Resolution taken in this action (like tabled, decided...)")
+    )
     case = models.ForeignKey(Case, related_name='actions', help_text=_('Case this action is related to'), blank=True,
                              null=True)
     ordering = models.IntegerField(help_text=_('Ordering of this action within a meeting'))
     article_number = models.CharField(max_length=255, blank=True,
                                       help_text=_('The article number given to this action after decision'))
-    resolution = models.CharField(max_length=255, blank=True,
```

Comment:
There are only a fixed number of different possible resolutions, so `resolution` should either be a non-translated CharField or a ForeignKey.